features/profile.feature

```diff
@@ -6,14 +6,21 @@ Feature: Profile page
   - Annotations are shown most recently updated first
 
   Background:
-    Given Josiah Carberry has 30 annotations
+    Given Josiah Carberry has 30 public annotations
 
-  Scenario: List shows 10 most-recently-updated annotations
+  Scenario: List shows 10 most-recently-updated public annotations on someone else's page
     When I go to Josiah Carberry's profile page
-    Then I should see his 10 most-recently-updated annotations in the 'Annotations' list
+    Then I should see his 10 most-recently-updated public annotations in the 'Annotations' list
+
+  Scenario: List shows 10 most-recently-updated annotations on your own page
+    Given I am Josiah Carberry
+    And I have logged in
+    And I have 30 public and private annotations
```

Comment:
this `And` conflicts with the `Background` as the annotations have become both public and private; use a different name?

Reply:
Removed the `Background` instead.